compio-driver/src/driver_type.rs

```diff
@@ -0,0 +1,115 @@
+use std::sync::atomic::{AtomicU8, Ordering};
+
+const UNINIT: u8 = u8::MAX;
+const IO_URING: u8 = 0;
+const POLLING: u8 = 1;
+const IOCP: u8 = 2;
+
+static DRIVER_TYPE: AtomicU8 = AtomicU8::new(UNINIT);
```

Comment:
How about using the `atomic` crate and store `Atomic<DriverType>` directly?

Reply:
That requires bytemuck. Is that ok?

Reply:
Is there any disadvantage?

Reply:
Not really, just a little extra effort and one more direct dependency.

Reply:
Then it depends on your choice. I just think `AtomicU8` is less readable. If you don't like another direct dependency, it's OK.